packages/css/src/switch/switch.scss

```diff
@@ -0,0 +1,68 @@
+.amsterdam-switch {
+  display: inline-block;
+  font-family: var(--amsterdam-switch-font-family);
+  font-size: var(--amsterdam-switch-font-size-narrow);
+  line-height: var(--amsterdam-switch-line-height);
+  outline-offset: var(--amsterdam-switch-outline-offset);
+  overflow: visible;
+  position: relative;
+}
+
+.amsterdam-switch__input {
+  appearance: none;
+  margin-block: 0;
+  margin-inline: 0;
+  opacity: 0%; /* to hide native input field in older iOS */
+}
+
+.amsterdam-switch__toggle {
+  background-color: var(--amsterdam-switch-background-color);
+  border-radius: 50px;
+  box-sizing: border-box;
```

Comment:
This is sort of a reset, right? The convention is to wrap resets in a `reset` mixin, so we know what its function is